src/main/java/com/zaxxer/nuprocess/linux/ProcessEpoll.java

```diff
@@ -228,7 +203,7 @@ public boolean process()
                return true;
             }
 
-            throw new RuntimeException("Error waiting for epoll (" + errno + ")");
+            throw new RuntimeException("Error waiting for epoll, errno: " + errno);
```

Comment:
Just rewording this to make the way `errno` is communicated more consistent across errors